docs/dev/tools/common-errors.md

```diff
@@ -184,27 +184,13 @@ This can happen when there is a slow internet connection, when the module is try
 **Solution:**
 
 - Try using a faster internet connection.
-- If the problem persists, try increasing the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables in the module's JSON configuration file.
-  For example:
-
-  ```json {class="line-numbers linkable-line-numbers"}
-  {
-    "modules": [
-      {
-        ...
-        "env": {
-          "VIAM_MODULE_STARTUP_TIMEOUT": "6m30s",
-          "VIAM_RESOURCE_CONFIGURATION_TIMEOUT": "3m0s"
-        }
-      }
-    ]
-  }
-  ```
-
 - If you are the module author, consider packaging the module with required dependencies so they don't need to be downloaded on startup.
   For Python modules, you can package your module with dependencies by using the PyInstaller steps when [uploading your module](/operate/get-started/other-hardware/#upload-your-module).
+- If the problem persists, try setting the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables on your machine.
```

Comment:
I worry that if we say this without saying *how* to set the env vars, and then continue to link to the module config details page below which shows using them with the config, people may make the same assumption I did that they can be used within the config.
We could clarify here but maybe overkill depending on how long it takes for the eng fix to go in.
```suggestion
- If the problem persists, try setting the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables on your machine.
  The environment variables will not be picked up from the JSON configuration until after the module has already started up and configured, so you must set the variables either from your command line or in the `viam-server` service config file.
```

Reply:
Great point. Unfortunately this still implies that the JSON configuration works, but unfortunately it doesn't sound like that is the case right now.
What do you think of also deleting the sentence below that refers to the module config details page? That way, users will only see a reference to environment variables, and they won't be biased into thinking that they should use the (nonfunctional) JSON config to set them.

We can bring the link-containing sentence back once engineering makes the JSON config for env vars functional.

Reply:
```suggestion
- If the problem persists, try setting the `VIAM_MODULE_STARTUP_TIMEOUT` or `VIAM_RESOURCE_CONFIGURATION_TIMEOUT` environment variables on your machine using the `viam-server` service configuration file.
  To load new environment variable values, restart `viam-server` from the Viam app.
```
To avoid mentioning the tricksy JSON, and make sure users actually load the vars, I trimmed your language down and added an extra instruction.

Reply:
This suggestion seems to disregard Cheuk's suggested command line method though?

Reply:
I couldn't get the command line method to work on any of my deployments, so I'm not comfortable recommending it to users 🥵 . If you know a command line method that works, I'm happy to incorporate it, but Cheuk and I both felt like the bash CLI approach was a little too error-prone!